### PR TITLE
[TW-941] Refresh sending-requests/managing-environments.md

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Break up Environments page into three pages
- Add new pages to the left nav, update redirects, and fix links
- Update all content for organization, accuracy, and style
- Update screenshots

### CODE REVIEW

File: src/pages/docs/sending-requests/environments/environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+## Set environment variables from scripts
+
+You can change the values of environment variables from your **Pre-request Script** and **Tests** scripts. Use the [`pm.environment` method](/docs/writing-scripts/script-references/postman-sandbox-api-reference/#using-environment-variables-in-scripts) to set an environment variable in the [active environment](/docs/sending-requests/environments/managing-environments/#switch-between-environments):
+
+```js
+pm.environment.set("variable_key", "variable_value");
+```
+
+If you use scripts to set values for environment variable, these values will be reflected in all requests that reference the variables. For example, you can use environments in conjunction with [the collection runner](/docs/collections/running-collections/intro-to-collection-runs/) and [monitors](/docs/monitoring-your-api/intro-monitors/) to use updated values for a series of requests throughout a run and after the run completes.
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/environments/environment-variables.md:86]:
```suggestion
If you use scripts to set values for environment variables, these values will be reflected in all requests that reference the variables. For example, you can use environments in conjunction with [the collection runner](/docs/collections/running-collections/intro-to-collection-runs/) and [monitors](/docs/monitoring-your-api/intro-monitors/) to use updated values for a series of requests throughout a run and after the run completes.
```

File: src/pages/docs/sending-requests/environments/team-environments.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+1. Enter the name, group name, or email address you want to assign a role to.
+1. Select the role you want your collaborators to have:
+
+    * **Editor** - Users can manage and edit the environment.
+    * **Viewer** - Users can view and share the environment.
+    * **Remove** - Users will no longer have access to the environment.
+
+1. Select **Update Roles**.
+
+### Use an environment with Viewer role
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/environments/team-environments.md:114]:
```suggestion
### Use an environment with the Viewer role
```

File: src/pages/docs/sending-requests/environments/team-environments.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+If you have the Viewer role for an environment, you can access the values of all variables in the environment. You can update a variable's current value, which is used to send requests in your local Postman instance and isn't shared with your team or synced to your Postman account. You can't edit the environment or update a variable's initial value, which is synced to the workspace and shared with your team.
+
+If you have Viewer access to an environment, a lock icon <img alt="Lock icon" src="https://assets.postman.com/postman-docs/icon-lock.jpg#icon" width="11px"> displays next to the environment's name to indicate that you can't make changes to the environment. If you need to edit the environment or initial values for variables, [request Editor access to the environment](#request-access-to-an-environment).
+
+### Use an environment with Editor role
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/environments/team-environments.md:120]:
```suggestion
### Use an environment with the Editor role
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM! Made just a few suggestions. Love the emphasis font format, not sure if you added it, but it's effective.

**Review by jkonrath-postman — APPROVED** [review]
LGTM! This is a great split and makes this much easier to understand!

**Pull request merged**